spec/catalog.yaml

```diff
@@ -102,7 +102,40 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/namespaces/{ns}:
+    parameters:
+      - $ref: '#/components/parameters/namespace'
+
+    get:
+      tags: [ Namespace ]
+      summary: Load the metadata properties for a namespace
+      operationId: loadNamespaceMetadata
+      description: Return all stored metadata properties for a given namespace
```

Comment:
a small comment for both for the `description` and also the `summary` above: in IRC we were thinking about adding more APIs to the namespace that is why the `loadNamespaceMetadata` name and related description is used, and that is also why there is a dedicated API for just updating properties at `/v1/{prefix}/namespaces/{namespace}/properties`. But now I think it's pretty clear at this moment that there won't be anything else than properties for a namespace... so I think we can safely just say the description is something more generic like "Get information about a namespace". What do you think?

Reply:
reasonable